pages/guides/developing/listen-for-webhooks.en-US.mdx

```diff
@@ -33,7 +33,7 @@ to a URL that you specify.
 | `multistream.error`        | This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting.                              |
 | `multistream.disconnected` | This fires when we are no longer sending video to the multistream target.                                                                                              |
 | `asset.created`            | This fires when a On Demand asset is created.                                                                                                                          |
-| `asset.updated`            | This fires when a On Demand asset is updated.                                                                                                                          |
+| `asset.updated`            | This fires when a On Demand asset is updated. The asset payload will contain a playback URL when playback is ready.                                                    |
```

Comment:
@victorges what do you think about this, is it worth adding or should we write something totally separate about the availability of playback?

Reply:
I think it's worth a short note here even if we have a separate explanation about it elsewhere. I'd just suggest a nit here, as `ready` looks like something that is finalized and gets confusing with the explicit `asset.ready` hook. WDYT of "available" instead?
```suggestion
| `asset.updated`            | This fires when a On Demand asset is updated. The asset payload will contain a playback URL when playback is available.                                                    |
```